crates/biome_css_formatter/src/css/any/value.rs

```diff
@@ -9,12 +9,13 @@ impl FormatRule<AnyCssValue> for FormatAnyCssValue {
     fn fmt(&self, node: &AnyCssValue, f: &mut CssFormatter) -> FormatResult<()> {
         match node {
             AnyCssValue::CssIdentifier(node) => node.format().fmt(f),
+            AnyCssValue::CssCustomIdentifier(node) => node.format().fmt(f),
+            AnyCssValue::CssDashedIdentifier(node) => node.format().fmt(f),
             AnyCssValue::CssString(node) => node.format().fmt(f),
             AnyCssValue::CssNumber(node) => node.format().fmt(f),
             AnyCssValue::AnyCssDimension(node) => node.format().fmt(f),
             AnyCssValue::CssRatio(node) => node.format().fmt(f),
             AnyCssValue::AnyCssFunction(node) => node.format().fmt(f),
-            AnyCssValue::CssCustomProperty(node) => node.format().fmt(f),
```

Comment:
`CssCustomProperty` is now `CssDashedIdentifier` and just directly wraps the `IDENT` token, rather than wrapping around an entire `CssIdentifier` node. This seems to be how most other parsers are handling this as well (SWC, LightningCSS, etc.).